errtracker/errtracker_test.go

```diff
@@ -49,23 +49,31 @@ type ErrtrackerTestSuite struct {
 
 var _ = Suite(&ErrtrackerTestSuite{})
 
+var truePath string
```

Comment:
You can write this as:
```
var truePath = osutil.FindInPathOrDefault("true", "/bin/true")
var falsePath = osutil.FindInPathOrDefault("false", "/bin/false")
```
to safe some lines of code :) (i.e. no need for the extra init() AFAICS).